### DISCUSSION TITLE
Safari support

### DISCUSSION THREAD

**Discussion by mfulton26** [apollographql/apollo-client-devtools#908] [Ideas]
I'd like to use this tool while inspecting a WKWebView JSContext using Safari as this cannot be done via Chrome or Firefox.

e.g. I got `react-native-debugger` to work okay but I'd much rather use this tool directly in my browser while debugging a web app in a `react-native-webview` on iOS. (I already can while on Android because I can use the Chrome extension.)

**Comment by ThePatriczek** [comment]
I miss this feature either. 
I would expect package like [react-devtools](https://www.npmjs.com/package/react-devtools) which can be started standalone as server linked with script tag to the project.

<img width="803" alt="image" src="https://user-images.githubusercontent.com/29193319/182304135-bdabe1c8-8db5-47d7-ad8f-610f3d4e5d36.png">
